File: Lib/DotNet/aaaSoft.Net/aaaSoft.Net/Xiep/Handlers/XiepPingRequestHandler.cs
using System;
using System.Collections.Generic;
using System.Text;
using aaaSoft.Net.Xiep.Packages;
using aaaSoft.Net.Xiep.EventArgs;

namespace aaaSoft.Net.Xiep.Handlers
{
    /**
     * 心跳请求处理器
     * @author aaa
     */
    public class XiepPingRequestHandler : AbstractRequestHandler
    {

        public override ResponsePackage execute(ClientConnectionInfoArgs clientConnectionInfoArgs, RequestPackage requestPackage)
        {
            ResponsePackage responsePackage = new ResponsePackage();
            responsePackage.Response = "XiepPong";
            return responsePackage;
        }
    }
}

File: trunk/Lib/Java/com.scbeta.net/src/com/scbeta/net/xiep/handlers/XiepPingRequestHandler.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.scbeta.net.xiep.handlers;

import com.scbeta.net.xiep.eventArgs.ClientConnectionInfoArgs;
import com.scbeta.net.xiep.packages.RequestPackage;
import com.scbeta.net.xiep.packages.ResponsePackage;

/**
 * 心跳请求
 * @author aaa
 */
public class XiepPingRequestHandler extends AbstractRequestHandler {

    @Override
    public ResponsePackage execute(ClientConnectionInfoArgs clientConnectionInfoArgs, RequestPackage requestPackage) {
        ResponsePackage responsePackage = new ResponsePackage();
        responsePackage.setResponse("XiepPong");
        return responsePackage;
    }
}
